### PR TITLE
Us1 b

### PR DESCRIPTION
**Pull request by fturello**

### CODE REVIEW

File: backend/src/controllers/userControllers.js [hunk 1]
```diff
@@ -17,18 +18,44 @@ const addUser = (req, res) => {
...
+//     .update(user)
+//     .then(([result]) => {
+//       res.location(`/user/${result.insertId}`).sendStatus(201);
+//     })
+//     .catch((err) => {
+//       console.error(err);
+//       res.sendStatus(500);
+//     });
+// };
+
```
Comment (lchastanet) [inline backend/src/controllers/userControllers.js]: Code commented

File: backend/src/models/UserManager.js [hunk 2]
```diff
@@ -7,32 +7,47 @@ class UserManager extends AbstractManager {
...
+  //       user.firstName,
+  //       user.lastName,
+  //       user.userName,
+  //       user.email,
+  //       user.password,
+  //       user.avatar_id,
+  //       user.id,
+  //     ]
+  //   );
+  // }
```
Comment (lchastanet) [inline backend/src/models/UserManager.js]: Code commented

File: backend/src/router.js [hunk 3]
```diff
@@ -4,13 +4,15 @@ const router = express.Router();
...
 
 const itemControllers = require("./controllers/itemControllers");
 const videoControllers = require("./controllers/videoControllers");
 const favoriteControllers = require("./controllers/favoriteControllers");
 const categoryControllers = require("./controllers/categoryControllers");
 const authorization = require("./middleware/authorization");
 const userControllers = require("./controllers/userControllers");
+const avatarController = require("./controllers/avatarController");
 
 // const verifyUser = require("./middleware/verifyUser");
```
Comment (lchastanet) [inline backend/src/router.js]: It remain commented

File: frontend/src/styles/Home.module.scss [hunk 4]
```diff
@@ -176,4 +226,15 @@ body {
...
+
+@media screen and (max-width: 1600px) {
+  .popular-images-container {
+    display: none;
+  }
+
+  // .main {
+  //   justify-content: center;
+  //   margin-left: -37rem;
+  // }
```
Comment (lchastanet) [inline frontend/src/styles/Home.module.scss]: Code commented

**Review by lchastanet — CHANGES_REQUESTED** [review]

**Review by lchastanet — APPROVED** [review]

**Comment by fturello** [comment]
@lchastanet

**Review by Julien879465 — APPROVED** [review]

**Pull request merged**